docs/organising-your-data/relationships.md

```diff
@@ -14,25 +14,38 @@ organisation to the individual.
 
 ## Creating relationships between contacts
 
-![image](/img/4.5_Add_Relationship.png)
+![image](/img/5.5_Add_Relationship.png)
 
 1.  Navigate to one of the records that you want to relate.
 2.  Click on **Add Relationship** from the **Actions** button on the
     contact summary, or from the Relationships tab
-3.  Select the Relationship Type. In this case it would be either "Child
-    of" or "Parent of".
+3.  Select the Relationship Type. In this case it would be either "Sibling
+    of".
```

Comment:
If there's one option we should remove the word "either".

Reply:
removed